### PR TITLE
Add all Beanstalk outputs

### PR DESCRIPTION
**Pull request by johncblandii**
## Feature

Some Beanstalk outputs were missing.

## Solution

Added all available Beanstalk outputs.

### CODE REVIEW

**Review by osterman — DISMISSED** [review]

File: outputs.tf [hunk 1]
```diff
@@ -27,3 +32,63 @@ output "ec2_instance_profile_role_name" {
...
+  value       = "${aws_elastic_beanstalk_environment.default.tier}"
+}
+
+output "application" {
+  description = "The Elastic Beanstalk Application specified for this environment."
+  value       = "${aws_elastic_beanstalk_environment.default.application}"
+}
+
+output "setting" {
+  description = "Settings specifically set for this Environment."
```
Comment (osterman) [inline outputs.tf]: Consider consistent capitalization for "Environment"

File: docs/terraform.md [hunk 2]
```diff
@@ -61,10 +61,23 @@
...
+| cname | Fully qualified DNS name for the Environment. |
+| description | Description of the Elastic Beanstalk Environment. |
 | ec2_instance_profile_role_name | Instance IAM role name |
 | elb_dns_name | ELB technical host |
 | elb_zone_id | ELB zone id |
 | host | DNS hostname |
+| id | ID of the Elastic Beanstalk Environment. |
+| instances | Instances used by this environment. |
+| launch_configurations | Launch configurations in use by this environment. |
+| load_balancers | Elastic load balancers in use by this environment. |
```
Comment (osterman) [inline docs/terraform.md]: Elastic Load Balancers (capitalization)

File: outputs.tf (hunk 1 above)
Comment (johncblandii) [reply outputs.tf]: That was copy/pasta. Good catch, though. Adjusting.

**Review by ESBA-salman — APPROVED** [review]
+1

**Pull request merged**